Xamarin.Forms.Platform.Android/Extensions/AccessibilityExtensions.cs

```diff
@@ -102,20 +103,21 @@ public static string SetNavigationContentDescription(this global::Android.Suppor
 
 		public static void SetTitleOrContentDescription(this IMenuItem Control, ToolbarItem Element)
 		{
-			if (Element == null)
-				return;
+			SetTitleOrContentDescription(Control, (MenuItem)Element);
+		}
```

Comment:
The main reason I left it in was for the possibilities of an ABI break. I realize that technically I don't need it but it would change the public api so I left it in.